app/models/queries/work_packages/columns/relation_of_type_column.rb

```diff
@@ -28,9 +28,8 @@
 
 class Queries::WorkPackages::Columns::RelationOfTypeColumn < Queries::WorkPackages::Columns::RelationColumn
   def initialize(type)
-    super
-
     self.type = type
+    super(name)
```

Comment:
Is it an unrelated bug fix?

Reply:
It relates to https://github.com/opf/openproject/pull/14440#discussion_r1471816323